go/vt/vtgate/router.go

```diff
@@ -51,6 +51,19 @@ func (rtr *Router) Execute(ctx context.Context, sql string, bindVars map[string]
 	return plan.Instructions.Execute(vcursor, queryConstruct, make(map[string]interface{}), true)
 }
 
+func (rtr *Router) ExecuteShow(ctx context.Context, sql string, bindVars map[string]interface{}, keyspace string, tabletType topodatapb.TabletType, session *vtgatepb.Session, notInTransaction bool, options *querypb.ExecuteOptions) (*sqltypes.Result, error) {
```

Comment:
I think this function is dead code. The call goes like this: VTGate.Execute->Router.Execute->Show.Execute->QueryExecutor.Execute. You should be able to just delete this and everything should still work.

The current relationship between vtgate and vtgate/engine is very confusing. We need to figure out a better model to represent the relationship between those two code groups.